solutions/swb-reference/cypress.config.ts

```diff
@@ -0,0 +1,41 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    fixturesFolder: 'end-to-end-tests/cypress/fixtures',
+    fileServerFolder: 'end-to-end-tests',
+    screenshotsFolder: 'end-to-end-tests/cypress/screenshots',
+    videosFolder: 'end-to-end-tests/cypress/videos',
+    downloadsFolder: 'end-to-end-tests/cypress/downloads',
+    supportFile: 'end-to-end-tests/cypress/support/e2e.ts',
+    specPattern: 'end-to-end-tests/cypress/**/*.cy.{js,jsx,ts,tsx}',
+    experimentalSessionAndOrigin: true,
+    setupNodeEvents(on, config) {
+      //this function is called in a different scope making cypress and cy commands unable to work
+      const environment = getEnvironmentVariables(config.env.STAGE);
+      environment.AdminPassword = config.env.AdminPassword ?? environment.AdminPassword; //read password from yaml only when password is not set in env variables already
+      config.env = { ...config.env, ...environment };
```

Comment:
To make the code easier to debug we should specify exactly what values E2E test needs instead of passing all config value in. This way if we change the config files in the future, we won't break the E2E tests

Reply:
config changed to specify needed env variables